sources/osgShadow/ShadowMap.js

```diff
@@ -383,7 +384,7 @@ ShadowMap.prototype = MACROUTILS.objectLibraryClass( MACROUTILS.objectInherit( S
             this.initTexture();
 
             this._textureUnit = this._textureUnitBase + lightNumber;
-            this._texture.setLightUnit( lightNumber );
+            this._texture.setLightNumber( lightNumber );
```

Comment:
I dont understand why we need lightIndex in the argument of the function. https://github.com/cedricpinson/osgjs/pull/681/commits/d937402f5ef44932e66e4d19110c271f3b11c36a#diff-8cf8f17e8132314edcfa57e87eda7ab4R364